covid_xprize/validation/scenario_generator.ipynb

```diff
@@ -287,6 +288,19 @@
     "## Generate"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Handle US Virgin Islands\n",
+    "LATEST_DATA_URL = 'https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv'\n",
+    "PHASE1_FILE = \"../../countries_regions_phase1_fix.csv\"\n",
+    "latest_df = load_dataset(LATEST_DATA_URL, PHASE1_FILE)\n",
+    "latest_df = phase1_update(latest_df)"
```

Comment:
The call to `phase1_update(latest_df)` makes the latest dataframe compatible with Phase 1.